api/BeforeUnloadEvent.json

```diff
@@ -36,6 +36,9 @@
           },
           "webview_android": {
             "version_added": true
+          },
+          "samsunginternet_android": {
+            "version_added": true
```

Comment:
Stuff like this is makes me wary since it shows  we are adding missing information.